airbyte-config/models/src/main/resources/types/StandardNormalizationSummary.yaml

```diff
@@ -0,0 +1,15 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardNormalizationSummary.yaml
+title: StandardNormalizationSummary
+description: standard information output by ALL sources for a normalization step
```

Comment:
Whether or not we run normalization depends on the destination rather than the source, so this description is a little bit confusing. Could you reword this to indicate that this information will only exist for syncs in which normalization was performed.